src/hmem.c

```diff
@@ -55,16 +56,20 @@ static struct ofi_hmem_ops hmem_ops[] = {
 		.cleanup = ofi_hmem_cleanup_noop,
 		.copy_to_hmem = ofi_memcpy,
 		.copy_from_hmem = ofi_memcpy,
+		.is_hmem_iface = ofi_is_hmem_iface,
```

Comment:
Ideally, this stand-alone function should fail if someone calls it with a GPU address, but I see you skip FI_HMEM_SYSTEM in ofi_get_hmem_iface().

Reply:
The problem here is that the address is likely a valid address for system memory.  (Is there a way to determine if a specific virtual address has been assigned to a process?  I'm not aware of one, but if so, maybe that could be part of some assert().)  We should capture this detail as a comment in the call below, since it's not obvious by reading the code why HMEM_SYSTEM is being skipped.  That detail would be nice as part of the commit message as well. :)